src/content/app/tools/blast/components/blast-input-sequences/useBlastInputSequences.test.tsx

```diff
@@ -0,0 +1,254 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { type ReactNode } from 'react';
+import useBlastInputSequences from './useBlastInputSequences';
+import { Provider } from 'react-redux';
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
+import { renderHook } from '@testing-library/react-hooks';
+
+import {
+  getSequences,
+  getSelectedSequenceType,
+  getSequenceSelectionMode
+} from 'src/content/app/tools/blast/state/blast-form/blastFormSelectors';
+import blastFormReducer, {
+  initialState as initialBlastFormState,
+  type BlastFormState
+} from 'src/content/app/tools/blast/state/blast-form/blastFormSlice';
+
+const getWrapper = (
+  { state }: { state?: Partial<BlastFormState> } = { state: {} }
+) => {
+  const blastFormState = Object.assign({}, initialBlastFormState, state);
+  const rootReducer = combineReducers({
```

Comment:
Can this be move outside of the `getWrapper` function?

Reply:
✅